python-threatexchange/threatexchange/exchanges/impl/ncmec_api.py

```diff
@@ -407,6 +410,10 @@ def fetch_value_merge(
         old: t.Optional[api.NCMECEntryUpdate],
         new: t.Optional[api.NCMECEntryUpdate],
     ) -> t.Optional[api.NCMECEntryUpdate]:
+        """
+        DEPRECATED: This method will be removed in version 2.x.x.
+        The deletion logic has been moved directly to fetch_iter().
+        """
         assert new is not None, "fetch shouldn't do this"
         if new.deleted:
             return None
```

Comment:
Hmm, this is a real functionality we need to store somewhere. The task mentions changing the behavior of fetch(). See L434 on the new code for where this is handled in convert